pkg/sriov/operatorconfig.go

```diff
@@ -29,6 +29,56 @@ type OperatorConfigBuilder struct {
 	errorMsg  string
 }
 
+// NewOperatorConfigBuilder creates new instance of OperatorConfigBuilder.
+func NewOperatorConfigBuilder(apiClient *clients.Settings, nsname string) *OperatorConfigBuilder {
+	glog.V(100).Infof(
+		"Initializing new OperatorConfigBuilder structure with the following params: %s", nsname)
```

Comment:
Maybe it's worth to replace param with namespace.
Looking at the log - not clear that the param is nsname

Reply:
agree with @achuzhoy on this one.